### PR TITLE
I added feature blblalba

### PR DESCRIPTION
**Pull request by ralhei**
more details 
- did this
- did that

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # my new project
 - feature x1
-- feature x2
+- feature xyz
```
Comment (ralhei) [inline README.md]: very stupid stuff

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
 - some more feature 
 bla bla bla
+udibububidu
```
Comment (ralhei) [inline README.md]: hmm, I've never seen that language, can you translate that to english?

Comment (ralhei) [reply README.md]: oh, there is not such translation, what to do?

File: README.md (hunk 1 above)
Comment (fu) [reply README.md]: ohh noooo! :)

**Pull request merged**